### PR TITLE
Restore interests option in the Builder's Challenge form

### PR DESCRIPTION
**Pull request by johnshaughnessy**
The `mozilla-builders-application-2024` newsletter has been created so that we can separately send updates about the program and the application process.

## Description

Describe what this change does.

- [ ] I have manually tested this.
- [x] I have recorded this change in `CHANGELOG.md`.

### CODE REVIEW

File: birdbox/microsite/forms.py [hunk 1]
```diff
@@ -98,6 +98,17 @@ def __init__(self, *args, **kwargs):
 
 class BuildersChallengeForm(ContactFormBase):
     def __init__(self, *args, **kwargs):
+        interests = forms.MultipleChoiceField(
+            label=(""),
+            choices=(
+                (
+                    "mozilla-builders-application-2024",
```
Comment (stevejalim) [inline birdbox/microsite/forms.py:105]: Hi @johnshaughnessy - is Basket/Braze set up to deal with this new interest name?

**Comment by stevejalim** [comment]
@johnshaughnessy When does this change need to go live, please?

**Comment by johnshaughnessy** [comment]
At your convenience. We're ready for it whenever you have the time to deploy but we were also OK with launching the site without the separate mailing list. (We went live this morning.)

File: birdbox/microsite/forms.py (hunk 1 above)
Comment (johnshaughnessy) [reply birdbox/microsite/forms.py:105]: Yes

**Comment by stevejalim** [comment]
Made some tweaks, including adding a (better than original) a label on the optional field, because we need them for a11y. I'll ship them like this, but we can tweak them if need be

Screenshot 2024-06-04 at 09 30 11

**Pull request merged**